test/cctest/test_node_crypto_env.cc

```diff
@@ -0,0 +1,35 @@
+#include "crypto/crypto_context.h"
+#include "gtest/gtest.h"
+#include "node_options.h"
+#include "node_test_fixture.h"
+#include "openssl/err.h"
+
+using node::AtExit;
+using node::RunAtExit;
+using v8::Local;
+using v8::String;
+
+/*
+ * This test verifies that an object created by LoadBIO supports BIO_tell
+ * and BIO_seek, otherwise PEM_read_bio_PrivateKey fails on some keys
+ * (if OpenSSL needs to rewind pointer between pem_read_bio_key()
+ * and pem_read_bio_key_legacy() inside PEM_read_bio_PrivateKey).
+ */
+class NodeCryptoEnv : public EnvironmentTestFixture {};
+
+static void at_exit_callback(void* arg) {}
+
+TEST_F(NodeCryptoEnv, LoadBIO) {
+  v8::HandleScope handle_scope(isolate_);
+  Argv argv;
+  Env env{handle_scope, argv};
+  //  just put a random string into BIO
+  Local<String> key = String::NewFromUtf8(isolate_, "abcdef").ToLocalChecked();
+  node::crypto::BIOPointer bio(node::crypto::LoadBIO(*env, key));
+  ASSERT_EQ(BIO_seek(bio.get(), 2), 2);
+  ASSERT_EQ(BIO_tell(bio.get()), 2);
+  ASSERT_EQ(ERR_peek_error(), 0UL) << "There should not have left "
+                                      "any errors on the OpenSSL error stack\n";
+  AtExit(*env, at_exit_callback, nullptr);
+  RunAtExit(*env);
```

Comment:
These two (and the callback a few lines up) aren't really necessary for the test.

Reply:
Without these strings test-macOS: test/parallel/test-inspector-wait-for-connection.js failed. I don't know why.

Reply:
Do you have access to results of previous tests of this pull request?

Reply:
> test/parallel/test-inspector-wait-for-connection.js failed

That's almost certainly a flaky test and not related to this PR. You can ignore it, they're just a fact of life with test suites this big. That particular test has been known flaky before, see #30619.

Anyway, a few extra lines in a test are not so important. I'll start the real CI.

Reply:
Before I start the CI all over again, can you remove these lines? Cheers.

Reply:
> Before I start the CI all over again, can you remove these lines? Cheers.

Done)